Project.toml

```diff
@@ -17,6 +17,7 @@ IterativeSolvers = "42fd0dbc-a981-5370-80f2-aaf504508153"
 LiftedMaps = "d22a30c1-52ac-4762-a8c9-5838452405e0"
 LinearAlgebra = "37e2e46d-f89d-539d-b4ee-838fcccc9c8e"
 LinearMaps = "7a12625a-238d-50fd-b39a-03d52299707e"
+Plotly = "58dd65bb-95f3-509e-9936-c39a10fdeae7"
```

Comment:
Could you please remove this dependency?

Reply:
Made a forced update of my remote branch, removing the dependency.